specifyweb/frontend/js_src/lib/components/Interactions/InteractionDialog.tsx

```diff
@@ -173,14 +173,16 @@ export function InteractionDialog({
       type: 'PreparationSelectState',
       entries: prepsData.map((prepData) => ({
         catalogNumber: prepData[0],
-        taxon: prepData[1],
-        preparationId: prepData[2],
-        prepType: prepData[3],
-        countAmount: prepData[4],
-        loaned: f.parseInt(prepData[5] ?? undefined) ?? 0,
-        gifted: f.parseInt(prepData[6] ?? undefined) ?? 0,
-        exchanged: f.parseInt(prepData[7] ?? undefined) ?? 0,
-        available: Number.parseInt(prepData[8]),
+        coId: prepData[1],
```

Comment:
Replace with `collectionObjectId`. Please don't use contraction in this codebase in the future it's not always clear what was meant by it.

Case in point, do you have an idea of what our previous engineer meant by `LDLM` in `LDLM_TO_MYSQL`? No other references to it in the codebase or online